programs/voter-stake-registry/Cargo.toml

```diff
@@ -35,13 +35,13 @@ spl-governance = { git = "https://github.com/solana-labs/solana-program-library"
 spl-governance-addin-api = { git = "https://github.com/solana-labs/solana-program-library", rev = "4c0bc4c968d5d6feaee18b1c633c636f20d66b15" }
 
 # refer solana to the same version anchor-lang 0.20.1 refers to
-solana-program = "1.8.5"
+solana-program = "1.9.5"
```

Comment:
@ckamm I am unsure if we should specify a hard version of some caret/tilde https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html, hence I would leave it with hard i.e. = 1.9.5